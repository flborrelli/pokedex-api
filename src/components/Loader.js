import React from "react";

function Loader() {
  return (
    <>
    <div>
      <img className='loader' src="/images/pokeball.png" alt="loader"/>
    </div>
    <div className='mt-3 loading-text'>
      Loading...
    </div>
    </>
  );
}

export default Loader;
