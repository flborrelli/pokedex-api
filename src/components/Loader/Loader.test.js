import React from 'react';
import Loader from './Loader';
import { create } from 'react-test-renderer';

  test('should match with loader snapshot', () => {
    const loaderComponent = create(<Loader />);
    expect(loaderComponent.toJSON()).toMatchSnapshot();
  });