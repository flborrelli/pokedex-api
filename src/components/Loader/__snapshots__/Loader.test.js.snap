// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should match with loader snapshot 1`] = `
Array [
  <div>
    <img
      alt="loader"
      className="loader"
      src="/images/pokeball.png"
    />
  </div>,
  <div
    className="mt-3 loading-text"
  >
    Loading...
  </div>,
]
`;

exports[`should match with loader snapshot 1`] = `
Array [
  <div>
    <img
      alt="loader"
      className="loader"
      src="/images/pokeball.png"
    />
  </div>,
  <div
    className="mt-3 loading-text"
  >
    Loading...
  </div>,
]
`;
